integ/scripts/bash/rfdk-integ-e2e.sh

```diff
@@ -71,6 +71,12 @@ export INFRASTRUCTURE_DEPLOY_FINISH_TIME=$SECONDS
 
 # Pull the top level directory for each cdk app in the components directory
 for COMPONENT in **/cdk.json; do
+    # Invoke hook function if it is exported and name is defined in HOOK_FUNCTION variable
+    if [ -n "$HOOK_FUNCTION" ]  && [ "$(type -t $HOOK_FUNCTION)" == "function" ]
```

Comment:
How would you feel about using a more descriptive variable name? Say `PRE_COMPONENT_HOOK`?